[BUG] Builder workflow not detecting the previous version

**Describe the bug**
The builder workflows are failing and the reason why they are failing is the query to get the new value of the previous version:
``` console
CURRENT="$(bash build-scripts/product_version.sh)"
PREVIOUS=$(curl -s https://api.github.com/repos/wazuh/wazuh-indexer/releases | jq -r '.[].tag_name' | grep -vE 'alpha|beta|rc' | sed 's/^v//' | grep -E '^4\.[0-9]+\.[0-9]+$' | \
grep -v "^${CURRENT}$" | sort -V | awk -v current="$CURRENT" '$0 < current' | tail -n1)

echo "Current version: $CURRENT"
echo "Latest release detected: $PREVIOUS"
echo "previous_version=$PREVIOUS" >> $GITHUB_OUTPUT
```

Output:
``` console
CURRENT="$(bash build-scripts/product_version.sh)"
PREVIOUS=$(curl -s https://api.github.com/repos/wazuh/wazuh-indexer/releases | jq -r '.[].tag_name' | grep -vE 'alpha|beta|rc' | sed 's/^v//' | grep -E '^4\.[0-9]+\.[0-9]+$' | \
grep -v "^${CURRENT}$" | sort -V | awk -v current="$CURRENT" '$0 < current' | tail -n1)

echo "Current version: $CURRENT"
echo "Latest release detected: $PREVIOUS"
echo "previous_version=$PREVIOUS" >> $GITHUB_OUTPUT
shell: /usr/bin/bash -e {0}
jq: error (at <stdin>:1): Cannot index string with string "tag_name"
Current version: 5.0.0
Latest release detected: 
```

Error:
``` console
Run sudo bash build-scripts/indexer_node_install.sh 
Error: Version argument is required.
Error: Process completed with exit code 1.
```

Also, it was detected that the previous version in the Debian test is still 4.12.0 which should be substituted by 5.0.0

**To Reproduce**
Steps to reproduce the behavior:
1. See [workflow run ](https://github.com/wazuh/wazuh-indexer/actions/runs/16416123388/job/46382578542)

**Expected behavior**
- The expected previous version should be `5.0.0`
- Investigate the reason why the query sometimes fails

--- a/.github/workflows/5_builderpackage_indexer.yml
+++ b/.github/workflows/5_builderpackage_indexer.yml
@@ -459,7 +459,7 @@ jobs:
           sudo apt-get remove --purge wazuh-indexer -y
 
       - name: Test DEB package update stopping the indexer
-        if: ${{ matrix.distribution == 'deb' && (matrix.architecture == 'x64' || (needs.setup.outputs.previous_version >= '4.12' && matrix.architecture == 'arm64'))}}
+        if: ${{ matrix.distribution == 'deb' && needs.setup.outputs.previous_version >= '5.0' }}
         run: |
           sudo bash build-scripts/indexer_node_install.sh ${{ needs.setup.outputs.previous_version }}
           sudo systemctl stop wazuh-indexer
@@ -468,12 +468,11 @@ jobs:
           sudo apt-get remove --purge wazuh-indexer -y
 
       - name: Test DEB package update without stopping the indexer
-        if: ${{ matrix.distribution == 'deb' && (matrix.architecture == 'x64' || (needs.setup.outputs.previous_version >= '4.12' && matrix.architecture == 'arm64'))}}
+        if: ${{ matrix.distribution == 'deb' && needs.setup.outputs.previous_version >= '5.0' }}
         run: |
 
           sudo bash build-scripts/indexer_node_install.sh ${{ needs.setup.outputs.previous_version }}
           sudo DEBIAN_FRONTEND=noninteractive dpkg -i --force-confnew "artifacts/dist/${{ steps.package.outputs.name }}"
-
 
       - name: Upload artifact
         uses: actions/upload-artifact@v4

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -27,6 +27,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Migrate smoke tests from Allocator to docker [(#931)](https://github.com/wazuh/wazuh-indexer/pull/931)
 - Migrate builder workflows from [(#930)](https://github.com/wazuh/wazuh-indexer/pull/930)
 - Rename bumper workflow file [(#986)](https://github.com/wazuh/wazuh-indexer/pull/986)
+- Update previous version in debian workflow test [(#1041)](https://github.com/wazuh/wazuh-indexer/pull/1041)
 
 ### Deprecated
 -

